src/main/java/it/gov/pagopa/afm/calculator/service/CalculatorService.java

```diff
@@ -173,14 +173,14 @@ private Transfer createTransfer(
         .idChannel(bundle.getIdChannel())
         .onUs(this.getOnUsValue(taxPayerFee, bundle, paymentOption))
         .abi(bundle.getAbi())
+        .subThreshold(isSubThreshold(taxPayerFee))
         .build();
   }
 
   private Boolean getOnUsValue(long taxPayerFee, ValidBundle bundle, PaymentOption paymentOption) {
     boolean onusValue = false;
     // if PaymentType is CP and amount > threshold ---> calculate onus value
-    if (bundle.getPaymentType().equalsIgnoreCase("cp")
-        && taxPayerFee >= Long.parseLong(StringUtils.trim(amountThreshold))) {
+    if (bundle.getPaymentType().equalsIgnoreCase("cp") && !isSubThreshold(taxPayerFee)) {
```

Comment:
use `payment amount` instead of `taxPayerFee`

Reply:
done